src/main.js

```diff
@@ -1,14 +1,30 @@
 import Vue from 'vue';
+import VueHighlightJS from 'vue-highlight.js';
 import App from './App.vue';
 import router from './router';
 import store from './store';
-import { BootstrapVue, IconsPlugin } from 'bootstrap-vue';
+import {
+  BootstrapVue,
+  IconsPlugin
+} from 'bootstrap-vue';
 
+// Highlight.js languages (Only required languages)
```

Comment:
Trzeba to przenieść do komponentu code.vue. Main.js nie jest dobrym miejscem bo opozniasz pierwsze załadowanie strony. Main.js ma być tak mały jak tylko to możliwe.